### PR TITLE
Fixd some mistakes in Simplified Chinese translation

### PR DESCRIPTION
**Pull request by PerfBleu**
Fixd some mistakes in Simplified Chinese translation

### CODE REVIEW

File: po/zh_CN.po [hunk 1]
```diff
@@ -2598,7 +2598,7 @@ msgstr "磁盘"
 
 #: pkg/metrics/metrics.jsx:794
 msgid "Disks usage"
-msgstr "磁盘使用"
+msgstr "磁盘使用率"
```
Comment (martinpitt) [inline po/zh_CN.po:2601]: These both translate to the same "Disk usage" in Google translate. I copied this to https://translate.fedoraproject.org/translate/cockpit/main/zh_CN/ . We don't use PRs to manage translations, but that weblate instance.

File: po/zh_CN.po [hunk 2]
```diff
@@ -8501,16 +8501,16 @@ msgstr "重载"
 #: pkg/storaged/lvm2/physical-volume.jsx:119
 #: pkg/storaged/stratis/filesystem.jsx:226 pkg/storaged/stratis/pool.jsx:518
 msgid "Usage"
-msgstr "用法"
+msgstr "负载状态"
```
Comment (martinpitt) [inline po/zh_CN.po:8504]: [google translate](https://translate.google.de/?sl=auto&tl=en&text=%E7%94%A8%E6%B3%95&op=translate) says that the original word means "Usage", but your new proposal [means](https://translate.google.de/?sl=auto&tl=en&text=%E8%B4%9F%E8%BD%BD%E7%8A%B6%E6%80%81&op=translate) "Load status" -- that seems too specific to be a random Google translate error. Can you please double-check this?

File: po/zh_CN.po [hunk 3]
```diff
@@ -8501,16 +8501,16 @@ msgstr "重载"
...
 
 #: pkg/storaged/storage-controls.jsx:206
 msgid "Usage of $0"
-msgstr "$0 的用法"
+msgstr "$0 的占用率"
```
Comment (martinpitt) [inline po/zh_CN.po:8508]: This changes "usage" to "occupancy" in the English translation. This seems at least plausible, but not really consistent with the other translations?

**Review by martinpitt — COMMENTED** [review]
Thanks! Please see my individual comments.

**Comment by PerfBleu** [comment]
1. for the 2nd part, 用法 in Chinese means "how to use a thing", the usage here doesn't mean that definitely, I think the usage here means the overview of system load, which can be translate as "系统负载 (system load)", "占用率 (usage)" or "负载状态 (Occupancy rate/usage)", but 系统负载 should include many indicators instead of CPU and ram only, 占用率 has the same problem in this context. I think 负载状态 is good, because in Chinese when 负载 comes out individually, it specifically refers to CPU and ram usually.
(the 用法 is definitely wrong here)

2. for the 3rd part, the "usage", which means "how much of the resource is in use", usually translate as "使用率" or "占用率" in Chinese.

3. for the selection of 使用率 and 占用率 in the "usage of disk", 使用率 usually means the capacity usage, 占用率 usually means the I/o usage of the disk.

________________________________

**Comment by PerfBleu** [comment]
________________________________

File: po/zh_CN.po (hunk 2 above)
Comment (martinpitt) [reply po/zh_CN.po:8504]: > 1. for the 2nd part, 用法 in Chinese means "how to use a thing", the usage here doesn't mean that definitely, I think the usage here means the overview of system load, which can be translate as "系统负载 (system load)"

No, not at all! This is how much space of a storage device is currently being used. It's not at all related to the system load.

Comment (PerfBleu) [reply po/zh_CN.po:8504]: You sure? I think it displays here , the red circle. Could you please make sure about that?
I switched language to zh-TW, it shows "效能狀態", whose string locates the same section in zh-CN.po.
debian-in-windows-edge

Comment (martinpitt) [reply po/zh_CN.po:8504]: Right, it *also* shows there. See the headers in the po file. This is a very generic word which appears in lots of places:

```
#: pkg/systemd/overview-cards/usageCard.jsx:128 pkg/metrics/metrics.jsx:1964
#: pkg/metrics/metrics.jsx:1965 pkg/metrics/metrics.jsx:1966
#: pkg/metrics/metrics.jsx:1967 pkg/storaged/btrfs/device.jsx:80
#: pkg/storaged/btrfs/volume.jsx:129 pkg/storaged/btrfs/filesystem.jsx:79
#: pkg/storaged/filesystem/filesystem.jsx:152
#: pkg/storaged/block/unrecognized-data.jsx:50
#: pkg/storaged/lvm2/physical-volume.jsx:119
#: pkg/storaged/stratis/filesystem.jsx:226 pkg/storaged/stratis/pool.jsx:518
```

So translating it as "system load" would be wrong -- that has a [very specific meaning in Unix/Linux](https://www.howtogeek.com/194642/understanding-the-load-average-on-linux-and-other-unix-like-systems/) and doesn't apply at all to storage devices and other metrics.

If the concept of "Usage" in this generic form cannot be expressed in Chinese properly, gettext has a concept of "contexts", e.g. "Usage" in the context of storage devices or "Usage" in the context of CPUs. We don't use them often as they are hard to understand for translators and easy to get wrong; but we could if necessary.

**Comment by PerfBleu** [comment]
Oh, I see, then "占用率" is ok here, it's the generic expression
________________________________

File: po/zh_CN.po (hunk 2 above)
Comment (PerfBleu) [reply po/zh_CN.po:8504]: I just replied through the E-mail, that reply is not so complete, so I reply here again.
My bad, I didn't realize that string was used in so many places.
In generic situation, "usage" can be translate as "占用" or "占用率", the difference between these 2 words is that the "率" stands for "ratio", so if there's no ratio, "占用" is more suitable.

Comment (martinpitt) [reply po/zh_CN.po:8504]: Thanks for the explanation! I updated it to "占用率" then, as these are all ratios.

**Comment by martinpitt** [comment]
I applied some changes, see the threads above. Can you please review your other changes accordingly here, and either update it directly in https://translate.fedoraproject.org/translate/cockpit/main/zh_CN/?q=Usage or this PR? Thanks!

**Comment by martinpitt** [comment]
As I said, I applied most discussed things from above to weblate. The second and third commits don't work -- you  cannot really edit the *.po files, and even less so change their structure or introduce contexts. This needs to happen in the source. 
So there is nothing actionable remaining here. If you have further tweaks, please apply/suggest them in https://translate.fedoraproject.org/translate/cockpit/main/zh_CN/?q=Usage

Thank you!

**Pull request closed without merging**